webcommon/lib.chrome_devtools_protocol/nbproject/org-netbeans-lib-chrome_devtools_protocol.sig

```diff
@@ -1,5 +1,9 @@
 #Signature file v4.1
+<<<<<<< Updated upstream
 #Version 2.0
+=======
+#Version 1.2
+>>>>>>> Stashed changes
```

Comment:
fixed for what is in master

Reply:
Thank you and yes the version in master is correct. I merged the JS ChromeDevTool fixes into master and overlooked, that this would clash here. Sorry about that!
If I see this correctly, the file is not in the commit anymore, which looks right.